targets/PythonSdk/source/source/PlayFabHTTP.py

```diff
@@ -0,0 +1,68 @@
+import json
+import PlayFabSettings
+import PlayFabErrors
+import requests
+
+# Note this is a blocking call and will always run synchronously
+# the return type is a dictionary that should contain a valid dictionary that
+# should reflect the expected JSON response
+# if the call fails, there will be a returned PlayFabError
+def DoPost(urlPath, request, authKey, authVal, extraHeaders, callback):
```

Comment:
Swap the order of callback and extraHeaders.
Give extraHeaders a default value of None

Reply:
customData is missing, and should also default to None.
Final 3 parameters should be:
, callback, customData = None, extraHeaders = None
We also need to populate result.customData and error.customData in every place were we create a return obj.